### PR TITLE
Updated info.json

### PR DESCRIPTION
**Pull request by cs-rohit-khune**
- Marked SP as `uncertified`
- Updated recordSet content, added keys `name` and `apiName`
- Updated view template entries to the latest format

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "knowledgeBase",
     "version": "1.0.1",
     "type": "solutionpack",
-    "local": false,
+    "local": true,
```
Comment (cs-suyog-jinturkar) [inline info.json:5]: What does this means?

**Review by cs-suyog-jinturkar — COMMENTED** [review]
one small review comment

File: info.json (hunk 1 above)
Comment (cs-rohit-khune) [reply info.json:5]: It determines if the solution pack is locally developed(true) or deployed from the marketplace(false).

Comment (cs-rohit-khune) [reply info.json:5]: This however has no impact on solution pack execution, when SP is deployed via market place it automatically updates to false

Comment (cs-suyog-jinturkar) [reply info.json:5]: interesting...

**Pull request merged**